src/test/regress/sql/multi_alter_table_add_constraints.sql

```diff
@@ -239,6 +239,12 @@ SELECT create_distributed_table('products', 'product_no');
 -- Command below should error out since 'name' is not a distribution column
 ALTER TABLE products ADD CONSTRAINT exc_name EXCLUDE USING btree (name with =);
 
+-- check that we can disable the constraint check
+BEGIN;
+SET LOCAL citus.allow_unsafe_constraints TO on;
+ALTER TABLE products ADD CONSTRAINT exc_name EXCLUDE USING btree (name with =);
```

Comment:
for completeness, it might be nicer to have similar test where we define a primary key

Reply:
sure, added below